launch_ros/launch_ros/utilities/namespace_utils.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2020 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Module with utility functions for handling namespaces."""
+
+
+from typing import Optional
+from typing import Text
+
+
+def is_root_namespace(ns: Text) -> bool:
+    """Return `True` if `ns` is `'/'`."""
+    return ns == '/'
+
+
+def is_namespace_absolute(ns: Text) -> bool:
+    """Return `True` if `ns` is absolute."""
+    return ns.startswith('/')
+
+
+def prefix_namespace(
+    base_ns: Optional[Text],
+    ns: Optional[Text],
+    *,
+    return_absolute_ns: bool = True,
+) -> Optional[Text]:
+    """
+    Return `ns` prefixed with `base_ns` if `ns` is relative, returns `ns` if not.
+
+    :param `base_ns`: Prefix to be added to `ns`.
+    :param `ns`: Namespace to be prefixed.
+    :param `return_absolute_ns`: if `True`, the result will be made absolute.
+    :return: `None` if both `base_ns` and `ns` are `None`, or
+        `base_ns` if `ns` is `None`, or
+        `ns` if `base_ns` is `None`, or
+        `ns` if `ns` is absolute, or
+        `ns` prefixed with `base_ns`.
+        In all cases, trailing `/` are stripped from the result.
+
+    ## Examples:
+
+    ```python3
+    combined_ns = prefix_namespace('my_ns', 'original_ns', return_absolute_ns=True)
+    assert comined_ns == '/my_ns/original_ns'
```

Comment:
@ivanpauno nit: 
```suggestion
    assert combined_ns == '/my_ns/original_ns'
```
same elsewhere.

Reply:
Check the other examples below, they have the same defect.